apps/sync/index.html

```diff
@@ -11,4 +11,7 @@
        are still in development.
      </p>
   </body>
+  <script src="js/fxsync-webcrypto/stringconversion.js"></script>
+  <script src="js/fxsync-webcrypto/keyderivation.js"></script>
+  <script src="js/fxsync-webcrypto/fxsyncwebcrypto.js"></script>
```

Comment:
OK, I'll remove these lines from this patch, and then we can do lazy loading of all the scripts when we put the whole things together.